src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -10,6 +10,7 @@
     <IsTargetingNetCoreApp Condition="'$(TargetGroup)'=='netcoreapp' OR '$(TargetGroup)'=='uap' OR '$(TargetGroup)'=='uapaot'">true</IsTargetingNetCoreApp>
     <IsPartialFacadeAssembly Condition="'$(IsTargetingNetFx)'=='true' OR '$(IsTargetingNetCoreApp)'=='true'">true</IsPartialFacadeAssembly>
     <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.0'">netstandard1.0;portable-net45+win8+wp8+wpa81</PackageTargetFramework>
+    <DefineConstants Condition="'$(IsTargetingNetCoreApp)' != 'true'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
@jkotas @eerhardt this is the latest change to define `netcoreapp` so that the shared code brought in from dotnet coreclr repo would compile...

Reply:
This doesn't make sense to me.  This is saying "if we aren't targeting netcoreapp, define `netcoreapp`".

Why wasn't this compiling without this change?  What were the errors?

---
In reply to: [173233671](https://github.com/dotnet/corefx/pull/26499#discussion_r173233671) [](ancestors = 173233671)

Reply:
Please check [this reply](https://github.com/dotnet/corefx/pull/26499#discussion_r173238320)

Reply:
Between, wouldn't the above line be required to define the preprocessor constant `netcoreapp`? I thought the `TargetGroup` is different from the constant used inside the code... that is why I added that line...

Reply:
`TargetGroup` is different from the constant used in the code.  But what this line is saying is "when IsTargetingNetCoreApp is NOT true, then set the `netcoreapp` define".  That is not correct.

`'$(IsTargetingNetCoreApp)' != 'true'` => `'$(IsTargetingNetCoreApp)' == 'true'` would be the right fix.  But I don't think this line is necessary at all since we are not compiling the Vector.cs file in this .csproj when targeting netcoreapp.

Reply:
Done!